src/main/java/org/graylog/integrations/notifications/types/SlackEventNotification.java

```diff
@@ -154,27 +159,38 @@ private String buildMessageTitle(EventNotificationContext ctx) {
 
 	String buildCustomMessage(EventNotificationContext ctx, SlackEventNotificationConfig config, String template) throws PermanentEventNotificationException {
 		List<MessageSummary> backlog = getAlarmBacklog(ctx);
-		Map<String, Object> model = getCustomMessageModel(ctx, config, backlog);
-		try {
-			LOG.debug("template = {} model = {}" ,template, model);
-			return templateEngine.transform(template, model);
-		} catch (Exception e) {
-			LOG.error("Exception during templating [{}]", e.toString());
-			throw new PermanentEventNotificationException(e.toString(),e.getCause());
+		backlog = getMessageBacklog(config, backlog);
```

Comment:
Overwriting existing variables can be quite confusing and can easily lead to errors. Please create a new variable to store the modified backlog. (e.g. `truncatedBacklog`)

BTW, that's one of the reasons why I usually mark every variable and field as `final` if possible. That way you can't overwrite existing variables by accident.